From 9b8ac2217743a522f0c4f3e360f089b434fcd04b Mon Sep 17 00:00:00 2001
From: Hal Feng <hal.feng@starfivetech.com>
Date: Tue, 21 Jan 2025 11:42:35 +0800
Subject: [PATCH 16/55] riscv: Fix __memcpy_aligned alias

Don't set the weak global alias of memcpy_aligned to memcpy.
This affected iperf3 test.

Signed-off-by: Hal Feng <hal.feng@starfivetech.com>
---
 arch/riscv/lib/memcpy_aligned.S | 1 -
 1 file changed, 1 deletion(-)

--- a/arch/riscv/lib/memcpy_aligned.S
+++ b/arch/riscv/lib/memcpy_aligned.S
@@ -80,6 +80,5 @@ SYM_FUNC_START(__memcpy_aligned)
 6:
 	ret
 SYM_FUNC_END(__memcpy_aligned)
-SYM_FUNC_ALIAS_WEAK(memcpy, __memcpy_aligned)
 SYM_FUNC_ALIAS(__pi_memcpy, __memcpy_aligned)
 SYM_FUNC_ALIAS(__pi___memcpy, __memcpy_aligned)
